hw3/exercise03/src/pages/Navbar.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Search from "./Search";
+import Houses from "./Houses";
+import Home from "./Home";
+
+export const Navbar = (props) => {
+  const { gameOfThronesData } = props;
+  return (
+    <>
+      <nav className="navbar navbar-expand text-align-center bg-dark">
+        <ul className="navbar-nav d-flex flex-wrap">
+          <li className="nav-item p-2">
+            <a className="nav-link text-white" href="/">
+              Home{" "}
+            </a>
+          </li>
+          <li className="nav-item p-2">
+            <a className="nav-link text-white" href="/houses">
+              Houses
+            </a>
+          </li>
+          <li className="nav-item p-2">
+            <a className="nav-link text-white" href="/search">
+              Search
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <BrowserRouter>
```

Comment:
Can also do the routing in App.js. makes the code a bit cleaner and easier to manage. 
```javascript
<BrowserRouter basename="/">
      <Routes>
          <Route exact path="/" element={<Home />} />
          <Route path="/search" element={<Search />} />
          <Route path="/houses" element={<Houses />} />
      </Routes>
</BrowserRouter>
```

Reply:
Funnily enough for some reason I couldn't get it to work in App.js for some reason. Was one of the reasons I ended up putting it into the Navbar. Will continue investigating how to get it to work right in App.js though!